### PR TITLE
Update LBS API calls

### PR DESCRIPTION
**Pull request by gsvr**
The flows are as follows:
- MTN and Vodacom: log a request to `requestlookup` with the search and response template --> log a request to `lbsrequest` with the requestlookup data
- All others: locate with LocationState --> log coordinates to `requestlocation` --> log a request to `requestlookup` with the search and response template and location

The `search` format is (values are strings):

```
{
   "mmc": "true"
}
```

The response format is (values are strings):

```
{
    "type": "SMS",
    "to_addr": "27123",
    "template": "Your ClinicFinder results are: {{ results }}. Thank you for using the service."
}
```

---

The `requestlocation` call is as follows:

```
{
    "point": {
        "type": "Point",
            "coordinates": [x, y]
    }
}
```

---

The `requestlookup` call is as follows:

```
{
    "search": search,
    "response": response,
   "location": null or requestlocation
}
```

---

The `lbsrequest` call is as follows:

`{
    "search": {
        "msisdn": "27123"
   },
  "pointofinterest": requestlookup
}`

Note: msisdn must be for MTN or VodaCom, permission must be given, leading `+` should be stripped out.

### CODE REVIEW

**Comment by gsvr** [comment]
@imsickofmaps Could you please clarify that last sentence?

Also, what would the search format be if the user selected `nearest` clinic?  I'm assuming for `hct` it would be:
`{
    "hct": "true"
}`

**Comment by imsickofmaps** [comment]
`27712345679` not `+27123456789`

for nearest please submit all options:

```
{
   "hct":"true",
   "mmc": "true",
   "etc": "true"
}
```

**Comment by gsvr** [comment]
@imsickofmaps Ready for review.  Note that I had to merge in #1, so we should probably land that first.

File: src/app.js [hunk 1]
```diff
@@ -35,30 +52,60 @@ go.app = function() {
...
             var lookup_data = {
-                search: {
-                    mmc: ((self.im.user.answers.state_clinic_type === 'mmc') ? "true" : "false")
-                },
+                search: self.make_clinic_search_params(),
                 response: {
                     template_type: "SMS",
                     to_addr: contact.msisdn,
-                    template: "Your nearest clinic is %result%. Thanks for using Clinic Finder"
+                    template: "Your nearest clinic is %result%. Thanks for " +
```
Comment (imsickofmaps) [inline src/app.js]: should be `{{ results }}`

File: src/app.js [hunk 2]
```diff
@@ -35,30 +52,60 @@ go.app = function() {
...
+        self.manual_locate = function(contact) {
+            return self.http
+                .post(self.req_location_url, {
+                    data: self.make_location_data(contact)
+                })
+                .then(function(response) {
+                    return self.http
+                        .post(self.req_lookup_url, {
+                            data: self.make_lookup_data(contact,
+                                self.req_location_url + response.data.id + '/')
```
Comment (imsickofmaps) [inline src/app.js]: just return `response.data`

**Comment by imsickofmaps** [comment]
:+1:

**Pull request merged**